packages/terra-collapsible-menu-view/tests/wdio/collapsible-menu-view-spec.js

```diff
@@ -1,64 +1,53 @@
   });
-
-  Terra.it.matchesScreenshot({ selector: '#root' });
-  Terra.it.isAccessible();
 });
 
 Terra.describeViewports('Collapsible Menu With Item Before Divider', ['tiny', 'small', 'medium', 'large'], () => {
-  before(() => {
-    browser.url('/#/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/default-collapsible-menu-view');
+  it('should display with item before divider', () => {
+    browser.url('/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/default-collapsible-menu-view');
     browser.click('[data-collapsible-menu-toggle]');
+    Terra.validates.element('default', { selector: '[data-terra-popup-content]' });
   });
-
-  Terra.it.matchesScreenshot({ selector: '[data-terra-popup-content]' });
-  Terra.it.isAccessible();
 });
 
 Terra.describeViewports('First hidden item visible when isIconOnly', ['medium'], () => {
-  before(() => browser.url('/#/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/first-hidden-item-is-icon'));
-
-  Terra.it.matchesScreenshot();
-  Terra.it.isAccessible();
+  it('should display First hidden item visible when isIconOnly', () => {
+    browser.url('/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/first-hidden-item-is-icon');
+    Terra.validates.element('default');
+  });
 });
 
 Terra.describeViewports('Responsive Single Item', ['tiny', 'large'], () => {
-  before(() => {
-    browser.url('/#/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/collapsible-menu-view-single-item');
+  it('should display responsive single item', () => {
+    browser.url('/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/collapsible-menu-view-single-item');
     browser.moveToObject('#root', 0, 0);
+    Terra.validates.element('default', { selector: '#root' });
   });
 
   Terra.describeViewports('Single Item Group', ['enormous', 'large', 'huge', 'medium'], () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/single-item-group');
+    it('should display single item group', () => {
+      browser.url('/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/single-item-group');
+      Terra.validates.element('default');
     });
-
-    Terra.it.matchesScreenshot();
-    Terra.it.isAccessible();
   });
 
   Terra.describeViewports('Single Item Group Hidden Open', ['tiny', 'small'], () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/single-item-group');
+    it('should display single item group hidden open', () => {
+      browser.url('/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/single-item-group');
       browser.click('[data-collapsible-menu-toggle]');
+      Terra.validates.element('default', { selector: '#root' });
     });
-
-    Terra.it.matchesScreenshot({ selector: '#root' });
-    Terra.it.isAccessible();
   });
-  Terra.it.matchesScreenshot({ selector: '#root' });
-  Terra.it.isAccessible();
```

Comment:
any idea what the last screenshot was for?

Reply:
It was for responsive single item so ([here](https://github.com/cerner/terra-framework/pull/1313/files#diff-38d597d5f3fd119bcda706fe67a9ab57fa45b0d4e8ef7cec81165941b33205f3R33)) so moved it to the respective it block instead of at the end.